protoc-gen-go/testdata/grpc.pb.go

```diff
@@ -0,0 +1,370 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// source: grpc.proto
+
+/*
+Package testdata is a generated protocol buffer package.
+
+It is generated from these files:
+	grpc.proto
+
+It has these top-level messages:
+	SimpleRequest
+	SimpleResponse
+	StreamMsg
+	StreamMsg2
+*/
+package testdata
+
+import proto "github.com/golang/protobuf/proto"
+import fmt "fmt"
+import math "math"
+
+import (
+	context "golang.org/x/net/context"
```

Comment:
Committing this will cause golang/protobuf to have extra dependencies.
I created a [grpctests](https://github.com/gogo/grpctests) repo to keep these things separate and to avoid having any dependencies.

Reply:
This is in a `testdata` folder. It shouldn't matter right?

Reply:
Also, did you want me to move the tests into your grpctest package? Or for the repo owners here to create a test repo to mirror to gogo one?

Reply:
I'm not sure if I understand the intention behind separating the grpc dependency in tests to a different repository. Can you elaborate @awalterschulze? It seems like the transitive dependencies of gRPC aren't large in and of themselves, either.

Reply:
Currently golang/protobuf has one dependency in a test, which I specifically remove in gogo/protobuf.
It is nicer to depend on something that does not depend on anything except the standard library.
Any dependency is an extra dependency that needs to be managed.
Also even though this won't cause a circular dependency between packages, it does cause a circular dependency between repositories.
Its not the end of the world, but if we can avoid it, it does make things simpler.

Reply:
@paranoiacblack does your 👍 mean we should proceed to add this dependency, or that this test belongs in a different repo? I'm waiting to proceed on this.

Reply:
Oh actually, the 👍 comes from @puellanivis, so I guess there's some disagreement here on your side?